stack/docker-compose.yml

```diff
@@ -0,0 +1,94 @@
+name: default-services
+
+services:
+  postgres:
+    container_name: postgres
+    image: postgres:16.0-alpine3.18
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=admin
+      - POSTGRES_PASSWORD=admin
+    ports:
+      - "5432:5432"
+
+  pg-admin:
+    container_name: pg-admin
+    image: dpage/pgadmin4
+    environment:
+      - PGADMIN_DEFAULT_EMAIL=example@example.com
+      - PGADMIN_DEFAULT_PASSWORD=admin
```

Comment:
I suggest moving all env variables into .env files. If the env variables are only needed by a particular service we can create e.g. db.env, etc.

Reply:
dont know if an env file with 30 differen variables will help anyone or will make anything better

Reply:
I would encourage it because it decouples the static docker configuration from runtime relevant environment variables. This way if we want to change runtime stuff, we directly know we can look it up in the env files.